prow/jobs/jobs_config.yaml

```diff
@@ -21,6 +21,7 @@ aws_services:
 - sagemaker
 - sfn
 - sns
+- iam
```

Comment:
I think these are in alphabetical order?

Reply:
++ changed